service/test/_device_method_test.js

```diff
@@ -90,14 +90,14 @@ describe('DeviceMethod', function() {
       it('throws a ReferenceError if \'deviceId\' is \'' + badArg + '\'', function() {
         var method = new DeviceMethod({ methodName: 'foo', payload: null, responseTimeoutInSeconds: 42 }, {});
         assert.throws(function() {
-          method.invokeOnModule(badArg, 'moduleId', {}, function() {});
```

Comment:
>{}, [](start = 52, length = 3)

What were these doing here.  Nice catch.

Reply:
Won't fix.

---
In reply to: [222825769](https://github.com/Azure/azure-iot-sdk-node/pull/372#discussion_r222825769) [](ancestors = 222825769)